### ISSUE TITLE
Multiple "Dummy" created

### ISSUE THREAD

**Issue by chisarie** [stepjam/RLBench#113]
Hello, thank you very much for this nice library! 
I am currently trying to train on multiple environments, so I often need to do `env.get_task(....)`. The problem is, every time I do it a new "Dummy" is created in the simulation gui. Because of this, after a while I even get an error in terminal: `OSError: [Errno 28] No space left on device:`. Do you know how to solve this?
The multiple dummies can simply be reproduced by running:
```
from rlbench.environment import Environment
from rlbench.action_modes import ArmActionMode, ActionMode
from rlbench.observation_config import ObservationConfig
from rlbench.tasks import MT30_V1
import numpy as np


class Agent(object):

    def __init__(self, action_size):
        self.action_size = action_size

    def act(self, obs):
        arm = np.random.normal(0.0, 0.1, size=(self.action_size - 1,))
        gripper = [1.0]  # Always open
        return np.concatenate([arm, gripper], axis=-1)


obs_config = ObservationConfig()
obs_config.set_all(True)

action_mode = ActionMode(ArmActionMode.ABS_JOINT_VELOCITY)
env = Environment(
    action_mode, obs_config=obs_config, headless=False)
env.launch()

agent = Agent(env.action_size)

train_tasks = MT30_V1['train']

training_cycles_per_task = 30
training_steps_per_task = 80
episode_length = 40

for _ in range(training_cycles_per_task):

    task_to_train = np.random.choice(train_tasks, 1)[0]
    task = env.get_task(task_to_train)
    task.sample_variation()  # random variation

    for i in range(training_steps_per_task):
        if i % episode_length == 0:
            print('Reset Episode')
            descriptions, obs = task.reset()
            print(descriptions)
        action = agent.act(obs)
        obs, reward, terminate = task.step(action)

print('Done')
env.shutdown()
```

which is the example you provided, only with more iteration of environments. The result with multiple dummies looks like this
![Screenshot from 2021-03-08 18-29-50](https://user-images.githubusercontent.com/43339851/110358214-712b7a80-803c-11eb-9f48-8e2548719db1.png)

Thank you again for all the help :)

**Comment by stepjam** [comment]
Thanks for reporting. I'll take a look today and report back 👍

**Comment by stepjam** [comment]
This has now been fixed in version 1.1.0. 
I'm expecting to have 1.1.0 merged into main in the next few days.
